src/engine-actions/close-channels.js

```diff
@@ -0,0 +1,83 @@
+const {
+  listChannels,
+  closeChannel,
+  listPendingChannels
+} = require('../lnd-actions')
+/**
+ * Closes a channel given the channel point
+ *
+ * @param {String} paymentChannelNetworkAddress
+ * @param {String} fundingAmount - int64 string
+ * @returns {Promise<void>} returns void on success
+ */
+async function closeChannels ({ force = false, pending = false } = {}) {
+  const { channels = [] } = await listChannels({ client: this.client })
+  const activeChannels = channels.filter(c => c.active)
+
+  let inactiveChannels = []
+  if (force === true) {
```

Comment:
If specify force for `closeChannels` does this mean we only want to force close inactive channels? It sounds like what we actually want to do is `force` close ALL channels?